docs/public-api.md

```diff
@@ -0,0 +1,22 @@
+# Cursorless public API
```

Comment:
I would be tempted to add this one as a section to customization.md as well, because it is also a form of customization. If you prefer it as its own file, I'd at least leave a pointer to it somewhere in customization.md for discoverability

Reply:
done